src/components/Journey.js

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import { Button, FormControl } from "@mui/material";
+
+const style = {
+  position: "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: 400,
+  bgcolor: "background.paper",
+  border: "2px solid #000",
+  boxShadow: 24,
+  p: 4,
+};
+
+const Journey = ({ bookingInformation, bookingResponse }) => {
```

Comment:
Lets try to use same item as in core schema

Reply:
Updated